resources/views/posts/edit-add.blade.php

```diff
@@ -111,7 +111,14 @@
                             '_field_name'  => 'body',
                             '_field_trans' => get_field_translations($dataTypeContent, 'body')
                         ])
-                        <textarea class="form-control richTextBox" id="richtextbody" name="body" style="border:0px;">@if(isset($dataTypeContent->body)){{ $dataTypeContent->body }}@endif</textarea>
+                        @php
+                            $dataTypeRows = $dataType->{(isset($dataTypeContent->id) ? 'editRows' : 'addRows' )};
+                            $row = $dataTypeRows->where('field', 'body')->first();
```

Comment:
NOTE: This is *not* a DB call, it is just filtering the `Collection` to get the correct type.